### ISSUE TITLE
The learned transformation layers are not used when testing?

### ISSUE THREAD

**Issue by chmxu** [hytseng0509/CrossDomainFewShot#14]
Hi. 

It seems that the scale and shift are omitted in all feature transformation layers when test. 
[https://github.com/hytseng0509/CrossDomainFewShot/blob/master/methods/LFTNet.py#L189](https://github.com/hytseng0509/CrossDomainFewShot/blob/master/methods/LFTNet.py#L189)
[https://github.com/hytseng0509/CrossDomainFewShot/blob/master/methods/backbone.py#L207](https://github.com/hytseng0509/CrossDomainFewShot/blob/master/methods/backbone.py#L207). This means when testing, the proposed method is the same as the base model. Is this right?

**Comment by hytseng0509** [comment]
Yes, the feature-wise transformation layers are used only during the training phase to improve the model generalization.
